### PR TITLE
parse for job-cores and use that as EASYBUILD_PARALLEL

### PR DESCRIPTION
**Pull request by mboisson**
New PR to fix https://github.com/ComputeCanada/easybuild-computecanada-config/pull/61

### CODE REVIEW

File: eb [hunk 1]
```diff
@@ -127,6 +127,12 @@ export RSNT_EASYBUILD_MAGIC_COOKIE=263ca73bb634185aab1d1b41627fdbba
 # argument validation
 NEW_ARGS=()
 for argument in "$@"; do
+	if [[ $argument =~ ^--job-cores=[0-9]*$ ]]; then
+		kv=(${argument//=/ })
+		echo "Job-cores specified to ${kv[1]}, setting EASYBUILD_PARALLEL=${kv[1]}"
+		export EASYBUILD_PARALLEL=${kv[1]}
+		NEW_ARGS+=("$argument")
```
Comment (bartoldeman) [inline eb:134]: This could be double, done again for ebuser below...

**Review by bartoldeman — CHANGES_REQUESTED** [review]

File: eb (hunk 1 above)
Comment (mboisson) [reply eb:134]: I don't understand. We don't check `--job-cores` and we don't modify `EASYBUILD_PARALLEL` in the rest of the code ?

Comment (mboisson) [reply eb:134]: This check is simply done for everyone, not just for when it is run as `ebuser`

Comment (bartoldeman) [reply eb:134]: what I mean is that if run as `ebuser`, `NEW_ARGS+=("$argument")` is done twice, first time here (for everyone), and then again for `ebuser`

**Review by bartoldeman — APPROVED** [review]
lgtm

**Pull request merged**